identity-service/src/notifications/notificationQueue.js

```diff
@@ -251,13 +255,25 @@ async function drainPublishedMessages(logger, optimizelyClient) {
   return numProcessedNotifs
 }
 
-async function drainPublishedSolanaMessages(logger) {
+async function drainPublishedSolanaMessages(logger, optimizelyClient) {
   logger.info(
     `[notificationQueue:drainPublishedSolanaMessages] Beginning processing of ${pushNotificationQueue.PUSH_SOLANA_NOTIFICATIONS_BUFFER.length} notifications...`
   )
 
   let numProcessedNotifs = 0
   for (const bufferObj of pushNotificationQueue.PUSH_SOLANA_NOTIFICATIONS_BUFFER) {
+    const notificationMappingVar =
+      notificaitonTypeMapping[bufferObj.notification.type]
+    const isDisabled = getRemoteFeatureVarEnabled(
```

Comment:
could we fix the naming? does `getRemoteFeatureVarEnabled` return true if enabled? if so wouldn't we want the var to be `isEnabled`? since `isDisabled` as true would be misleading

Reply:
In identity, if it is true then it is disabled because the plugin also requests this var and if it is true in the plugin, then it starts sending out notifications. 
Can leave a comment

Reply:
oh duh sorry i missed that, thanks for explaining